Internal server error on post new page

### Describe the bug

When i go to the post creation page, as a guest user, i get a 500 error. 

### To Reproduce

1. As an admin visit admin area 
2. Identify the a space, enable / install a blog component . Publish it
3. Configure the component to allow users to create posts 
4. As admin, visit the newly created component on public site.  Click on Create post 
5. When the blog form is being visible, copy the url 
6. Open an incognito window and paste. 
7. You will see a 500 error 

### Expected behavior

When i visit a new blog post page, as unauthorized user, i should be redirected to login 


### Screenshots

<img width="2042" height="749" alt="Image" src="https://github.com/user-attachments/assets/632ec61f-0fdf-4988-ab35-636ac91e9f84" />

### Stacktrace

```
  
ActionView::Template::Error (undefined method &#39;name&#39; for nil)
Caused by: NoMethodError (undefined method &#39;name&#39; for nil)

Information for: ActionView::Template::Error (undefined method &#39;name&#39; for nil):
    1: &lt;div&gt;
    2:   &lt;%= fo_post_author_select_field form, :decidim_author_id, label: t(&quot;.author_id&quot;) %&gt;
    3: &lt;/div&gt;
    4: 
    5: &lt;div&gt;
  
/home/alecslupu/Sites/decidim/develop/decidim-blogs/app/helpers/decidim/blogs/posts_select_helper.rb:12:in &#39;Decidim::Blogs::PostsSelectHelper#fo_post_author_select_field&#39;
/home/alecslupu/Sites/decidim/develop/decidim-blogs/app/views/decidim/blogs/posts/_form.html.erb:2
actionview (8.1.3) lib/action_view/base.rb:281:in &#39;Kernel#public_send&#39;
actionview (8.1.3) lib/action_view/base.rb:281:in &#39;ActionView::Base#_run&#39;
actionview (8.1.3) lib/action_view/template.rb:284:in &#39;block in ActionView::Template#render&#39;
activesupport (8.1.3) lib/active_support/notifications.rb:212:in &#39;ActiveSupport::Notifications.instrument&#39;
actionview (8.1.3) lib/action_view/template.rb:589:in &#39;ActionView::Template#instrument_render_template&#39;
actionview (8.1.3) lib/action_view/template.rb:272:in &#39;ActionView::Template#render&#39;
actionview (8.1.3) lib/action_view/renderer/partial_renderer.rb:268:in &#39;block in ActionView::PartialRenderer#render_partial_template&#39;
activesupport (8.1.3) lib/active_support/notifications.rb:210:in &#39;block in ActiveSupport::Notifications.instrument&#39;
activesupport (8.1.3) lib/active_support/notifications/instrumenter.rb:58:in &#39;ActiveSupport::Notifications::Instrumenter#instrument&#39;
activesupport (8.1.3) lib/active_support/notifications.rb:210:in &#39;ActiveSupport::Notifications.instrument&#39;
actionview (8.1.3) lib/action_view/renderer/partial_renderer.rb:262:in &#39;ActionView::PartialRenderer#render_partial_template&#39;
actionview (8.1.3) lib/action_view/renderer/object_renderer.rb:31:in &#39;ActionView::ObjectRenderer#render_partial_template&#39;
actionview (8.1.3) lib/action_view/renderer/partial_renderer.rb:253:in &#39;ActionView::PartialRenderer#render&#39;
actionview (8.1.3) lib/action_view/renderer/object_renderer.rb:16:in &#39;ActionView::ObjectRenderer#render_object_with_partial&#39;
actionview (8.1.3) lib/action_view/renderer/renderer.rb:74:in &#39;ActionView::Renderer#render_partial_to_object&#39;
actionview (8.1.3) lib/action_view/renderer/renderer.rb:29:in &#39;ActionView::Renderer#render_to_object&#39;
actionview (8.1.3) lib/action_view/renderer/renderer.rb:24:in &#39;ActionView::Renderer#render&#39;
actionview (8.1.3) lib/action_view/helpers/rendering_helper.rb:145:in &#39;block in ActionView::Helpers::RenderingHelper#render&#39;
actionview (8.1.3) lib/action_view/base.rb:308:in &#39;ActionView::Base#in_rendering_context&#39;
actionview (8.1.3) lib/action_view/helpers/rendering_helper.rb:141:in &#39;ActionView::Helpers::RenderingHelper#render&#39;
/home/alecslupu/Sites/decidim/develop/decidim-blogs/app/views/decidim/blogs/posts/new.html.erb:11
actionview (8.1.3) lib/action_view/helpers/capture_helper.rb:50:in &#39;block in ActionView::Helpers::CaptureHelper#capture&#39;
actionview (8.1.3) lib/action_view/buffers.rb:75:in &#39;ActionView::OutputBuffer#capture&#39;
actionview (8.1.3) lib/action_view/helpers/capture_helper.rb:50:in &#39;ActionView::Helpers::CaptureHelper#capture&#39;
actionview (8.1.3) lib/action_view/helpers/form_helper.rb:775:in &#39;ActionView::Helpers::FormHelper#form_with&#39;
actionview (8.1.3) lib/action_view/helpers/form_helper.rb:462:in &#39;ActionView::Helpers::FormHelper#form_for&#39;
/home/alecslupu/Sites/decidim/develop/decidim-core/app/helpers/decidim/decidim_form_helper.rb:30:in &#39;Decidim::DecidimFormHelper#decidim_form_for&#39;
/home/alecslupu/Sites/decidim/develop/decidim-blogs/app/views/decidim/blogs/posts/new.html.erb:9
actionview (8.1.3) lib/action_view/helpers/capture_helper.rb:50:in &#39;block in ActionView::Helpers::CaptureHelper#capture&#39;
actionview (8.1.3) lib/action_view/buffers.rb:75:in &#39;ActionView::OutputBuffer#capture&#39;
actionview (8.1.3) lib/action_view/helpers/capture_helper.rb:50:in &#39;ActionView::Helpers::CaptureHelper#capture&#39;
actionview (8.1.3) lib/action_view/helpers/rendering_helper.rb:211:in &#39;ActionView::Helpers::RenderingHelper#_layout_for&#39;
actionview (8.1.3) lib/action_view/renderer/partial_renderer.rb:269:in &#39;block (2 levels) in ActionView::PartialRenderer#render_partial_template&#39;
/home/alecslupu/Sites/decidim/develop/decidim-core/app/views/layouts/decidim/shared/_layout_center.html.erb:14
actionview (8.1.3) lib/action_view/helpers/capture_helper.rb:50:in &#39;block in ActionView::Helpers::CaptureHelper#capture&#39;
actionview (8.1.3) lib/action_view/buffers.rb:75:in &#39;ActionView::OutputBuffer#capture&#39;
actionview (8.1.3) lib/action_view/helpers/capture_helper.rb:50:in &#39;ActionView::Helpers::CaptureHelper#capture&#39;
actionview (8.1.3) lib/action_view/helpers/tag_helper.rb:521:in &#39;ActionView::Helpers::TagHelper#content_tag&#39;
/home/alecslupu/Sites/decidim/develop/decidim-core/app/views/layouts/decidim/shared/_layout_center.html.erb:13
actionview (8.1.3) lib/action_view/base.rb:281:in &#39;Kernel#public_send&#39;
actionview (8.1.3) lib/action_view/base.rb:281:in &#39;ActionView::Base#_run&#39;
actionview (8.1.3) lib/action_view/template.rb:284:in &#39;block in ActionView::Template#render&#39;
activesupport (8.1.3) lib/active_support/notifications.rb:212:in &#39;ActiveSupport::Notifications.instrument&#39;
actionview (8.1.3) lib/action_view/template.rb:589:in &#39;ActionView::Template#instrument_render_template&#39;
actionview (8.1.3) lib/action_view/template.rb:272:in &#39;ActionView::Template#render&#39;
actionview (8.1.3) lib/action_view/renderer/partial_renderer.rb:268:in &#39;block in ActionView::PartialRenderer#render_partial_template&#39;
activesupport (8.1.3) lib/active_support/notifications.rb:210:in &#39;block in ActiveSupport::Notifications.instrument&#39;
activesupport (8.1.3) lib/active_support/notifications/instrumenter.rb:58:in &#39;ActiveSupport::Notifications::Instrumenter#instrument&#39;
activesupport (8.1.3) lib/active_support/notifications.rb:210:in &#39;ActiveSupport::Notifications.instrument&#39;
actionview (8.1.3) lib/action_view/renderer/partial_renderer.rb:262:in &#39;ActionView::PartialRenderer#render_partial_template&#39;
actionview (8.1.3) lib/action_view/renderer/partial_renderer.rb:253:in &#39;ActionView::PartialRenderer#render&#39;
actionview (8.1.3) lib/action_view/renderer/renderer.rb:78:in &#39;ActionView::Renderer#render_partial_to_object&#39;
actionview (8.1.3) lib/action_view/renderer/renderer.rb:49:in &#39;ActionView::Renderer#render_partial&#39;
actionview (8.1.3) lib/action_view/helpers/rendering_helper.rb:143:in &#39;block in ActionView::Helpers::RenderingHelper#render&#39;
actionview (8.1.3) lib/action_view/base.rb:308:in &#39;ActionView::Base#in_rendering_context&#39;
actionview (8.1.3) lib/action_view/helpers/rendering_helper.rb:141:in &#39;ActionView::Helpers::RenderingHelper#render&#39;
/home/alecslupu/Sites/decidim/develop/decidim-blogs/app/views/decidim/blogs/posts/new.html.erb:2
actionview (8.1.3) lib/action_view/base.rb:281:in &#39;Kernel#public_send&#39;
actionview (8.1.3) lib/action_view/base.rb:281:in &#39;ActionView::Base#_run&#39;
actionview (8.1.3) lib/action_view/template.rb:284:in &#39;block in ActionView::Template#render&#39;
activesupport (8.1.3) lib/active_support/notifications.rb:212:in &#39;ActiveSupport::Notifications.instrument&#39;
actionview (8.1.3) lib/action_view/template.rb:589:in &#39;ActionView::Template#instrument_render_template&#39;
actionview (8.1.3) lib/action_view/template.rb:272:in &#39;ActionView::Template#render&#39;
actionview (8.1.3) lib/action_view/renderer/template_renderer.rb:66:in &#39;block (2 levels) in ActionView::TemplateRenderer#render_template&#39;
activesupport (8.1.3) lib/active_support/notifications.rb:210:in &#39;block in ActiveSupport::Notifications.instrument&#39;
activesupport (8.1.3) lib/active_support/notifications/instrumenter.rb:58:in &#39;ActiveSupport::Notifications::Instrumenter#instrument&#39;
activesupport (8.1.3) lib/active_support/notifications.rb:210:in &#39;ActiveSupport::Notifications.instrument&#39;
actionview (8.1.3) lib/action_view/renderer/template_renderer.rb:60:in &#39;block in ActionView::TemplateRenderer#render_template&#39;
actionview (8.1.3) lib/action_view/renderer/template_renderer.rb:76:in &#39;block in ActionView::TemplateRenderer#render_with_layout&#39;
activesupport (8.1.3) lib/active_support/notifications.rb:210:in &#39;block in ActiveSupport::Notifications.instrument&#39;
activesupport (8.1.3) lib/active_support/notifications/instrumenter.rb:58:in &#39;ActiveSupport::Notifications::Instrumenter#instrument&#39;
activesupport (8.1.3) lib/active_support/notifications.rb:210:in &#39;ActiveSupport::Notifications.instrument&#39;
actionview (8.1.3) lib/action_view/renderer/template_renderer.rb:75:in &#39;ActionView::TemplateRenderer#render_with_layout&#39;
actionview (8.1.3) lib/action_view/renderer/template_renderer.rb:59:in &#39;ActionView::TemplateRenderer#render_template&#39;
actionview (8.1.3) lib/action_view/renderer/template_renderer.rb:11:in &#39;ActionView::TemplateRenderer#render&#39;
actionview (8.1.3) lib/action_view/renderer/renderer.rb:58:in &#39;ActionView::Renderer#render_template_to_object&#39;
actionview (8.1.3) lib/action_view/renderer/renderer.rb:31:in &#39;ActionView::Renderer#render_to_object&#39;
actionview (8.1.3) lib/action_view/rendering.rb:136:in &#39;block in ActionView::Rendering#_render_template&#39;
actionview (8.1.3) lib/action_view/base.rb:308:in &#39;ActionView::Base#in_rendering_context&#39;
actionview (8.1.3) lib/action_view/rendering.rb:135:in &#39;ActionView::Rendering#_render_template&#39;
actionpack (8.1.3) lib/action_controller/metal/streaming.rb:179:in &#39;ActionController::Streaming#_render_template&#39;
actionview (8.1.3) lib/action_view/rendering.rb:122:in &#39;ActionView::Rendering#render_to_body&#39;
actionpack (8.1.3) lib/action_controller/metal/rendering.rb:192:in &#39;ActionController::Rendering#render_to_body&#39;
actionpack (8.1.3) lib/action_controller/metal/renderers.rb:155:in &#39;ActionController::Renderers#render_to_body&#39;
actionpack (8.1.3) lib/abstract_controller/rendering.rb:28:in &#39;AbstractController::Rendering#render&#39;
actionpack (8.1.3) lib/action_controller/metal/rendering.rb:173:in &#39;ActionController::Rendering#render&#39;
actionpack (8.1.3) lib/action_controller/metal/instrumentation.rb:31:in &#39;block (2 levels) in ActionController::Instrumentation#render&#39;
activesupport (8.1.3) lib/active_support/benchmark.rb:17:in &#39;ActiveSupport::Benchmark.realtime&#39;
actionpack (8.1.3) lib/action_controller/metal/instrumentation.rb:31:in &#39;block in ActionController::Instrumentation#render&#39;
actionpack (8.1.3) lib/action_controller/metal/instrumentation.rb:100:in &#39;ActionController::Instrumentation#cleanup_view_runtime&#39;
activerecord (8.1.3) lib/active_record/railties/controller_runtime.rb:48:in &#39;ActiveRecord::Railties::ControllerRuntime#cleanup_view_runtime&#39;
actionpack (8.1.3) lib/action_controller/metal/instrumentation.rb:30:in &#39;ActionController::Instrumentation#render&#39;
actionpack (8.1.3) lib/action_controller/metal/implicit_render.rb:39:in &#39;ActionController::ImplicitRender#default_render&#39;
actionpack (8.1.3) lib/action_controller/metal/basic_implicit_render.rb:9:in &#39;ActionController::BasicImplicitRender#send_action&#39;
actionpack (8.1.3) lib/abstract_controller/base.rb:221:in &#39;AbstractController::Base#process_action&#39;
actionpack (8.1.3) lib/action_controller/metal/rendering.rb:199:in &#39;ActionController::Rendering#process_action&#39;
actionpack (8.1.3) lib/abstract_controller/callbacks.rb:267:in &#39;block in AbstractController::Callbacks#process_action&#39;
activesupport (8.1.3) lib/active_support/callbacks.rb:121:in &#39;block in ActiveSupport::Callbacks#run_callbacks&#39;
activesupport (8.1.3) lib/active_support/core_ext/time/zones.rb:65:in &#39;Time.use_zone&#39;
/home/alecslupu/Sites/decidim/develop/decidim-core/app/controllers/concerns/decidim/use_organization_time_zone.rb:21:in &#39;Decidim::ApplicationController#use_organization_time_zone&#39;
activesupport (8.1.3) lib/active_support/callbacks.rb:130:in &#39;block in ActiveSupport::Callbacks#run_callbacks&#39;
i18n (1.14.8) lib/i18n.rb:354:in &#39;I18n::Base#with_locale&#39;
/home/alecslupu/Sites/decidim/develop/decidim-core/app/controllers/concerns/decidim/locale_switcher.rb:25:in &#39;Decidim::ApplicationController#switch_locale&#39;
activesupport (8.1.3) lib/active_support/callbacks.rb:130:in &#39;block in ActiveSupport::Callbacks#run_callbacks&#39;
activesupport (8.1.3) lib/active_support/callbacks.rb:141:in &#39;ActiveSupport::Callbacks#run_callbacks&#39;
actionpack (8.1.3) lib/abstract_controller/callbacks.rb:266:in &#39;AbstractController::Callbacks#process_action&#39;
actionpack (8.1.3) lib/action_controller/metal/rescue.rb:36:in &#39;ActionController::Rescue#process_action&#39;
actionpack (8.1.3) lib/action_controller/metal/instrumentation.rb:76:in &#39;block in ActionController::Instrumentation#process_action&#39;
activesupport (8.1.3) lib/active_support/notifications.rb:210:in &#39;block in ActiveSupport::Notifications.instrument&#39;
activesupport (8.1.3) lib/active_support/notifications/instrumenter.rb:58:in &#39;ActiveSupport::Notifications::Instrumenter#instrument&#39;
activesupport (8.1.3) lib/active_support/notifications.rb:210:in &#39;ActiveSupport::Notifications.instrument&#39;
actionpack (8.1.3) lib/action_controller/metal/instrumentation.rb:75:in &#39;ActionController::Instrumentation#process_action&#39;
actionpack (8.1.3) lib/action_controller/metal/params_wrapper.rb:259:in &#39;ActionController::ParamsWrapper#process_action&#39;
activerecord (8.1.3) lib/active_record/railties/controller_runtime.rb:39:in &#39;ActiveRecord::Railties::ControllerRuntime#process_action&#39;
actionpack (8.1.3) lib/abstract_controller/base.rb:154:in &#39;AbstractController::Base#process&#39;
actionview (8.1.3) lib/action_view/rendering.rb:40:in &#39;ActionView::Rendering#process&#39;
actionpack (8.1.3) lib/action_controller/metal.rb:252:in &#39;ActionController::Metal#dispatch&#39;
actionpack (8.1.3) lib/action_controller/metal.rb:335:in &#39;ActionController::Metal.dispatch&#39;
actionpack (8.1.3) lib/action_dispatch/routing/route_set.rb:65:in &#39;ActionDispatch::Routing::RouteSet::Dispatcher#dispatch&#39;
actionpack (8.1.3) lib/action_dispatch/routing/route_set.rb:50:in &#39;ActionDispatch::Routing::RouteSet::Dispatcher#serve&#39;
actionpack (8.1.3) lib/action_dispatch/journey/router.rb:35:in &#39;block in ActionDispatch::Journey::Router#serve&#39;
actionpack (8.1.3) lib/action_dispatch/journey/router.rb:86:in &#39;block in ActionDispatch::Journey::Router#recognize&#39;
actionpack (8.1.3) lib/action_dispatch/journey/router.rb:66:in &#39;Array#each&#39;
actionpack (8.1.3) lib/action_dispatch/journey/router.rb:66:in &#39;ActionDispatch::Journey::Router#recognize&#39;
actionpack (8.1.3) lib/action_dispatch/journey/router.rb:31:in &#39;ActionDispatch::Journey::Router#serve&#39;
actionpack (8.1.3) lib/action_dispatch/routing/route_set.rb:906:in &#39;ActionDispatch::Routing::RouteSet#call&#39;
railties (8.1.3) lib/rails/engine.rb:534:in &#39;Rails::Engine#call&#39;
railties (8.1.3) lib/rails/railtie.rb:225:in &#39;Kernel#public_send&#39;
railties (8.1.3) lib/rails/railtie.rb:225:in &#39;Rails::Railtie.method_missing&#39;
actionpack (8.1.3) lib/action_dispatch/routing/mapper.rb:33:in &#39;block in &lt;class:Constraints&gt;&#39;
actionpack (8.1.3) lib/action_dispatch/routing/mapper.rb:62:in &#39;ActionDispatch::Routing::Mapper::Constraints#serve&#39;
actionpack (8.1.3) lib/action_dispatch/journey/router.rb:35:in &#39;block in ActionDispatch::Journey::Router#serve&#39;
actionpack (8.1.3) lib/action_dispatch/journey/router.rb:92:in &#39;block in ActionDispatch::Journey::Router#recognize&#39;
actionpack (8.1.3) lib/action_dispatch/journey/router.rb:66:in &#39;Array#each&#39;
actionpack (8.1.3) lib/action_dispatch/journey/router.rb:66:in &#39;ActionDispatch::Journey::Router#recognize&#39;
actionpack (8.1.3) lib/action_dispatch/journey/router.rb:31:in &#39;ActionDispatch::Journey::Router#serve&#39;
actionpack (8.1.3) lib/action_dispatch/routing/route_set.rb:906:in &#39;ActionDispatch::Routing::RouteSet#call&#39;
railties (8.1.3) lib/rails/engine.rb:534:in &#39;Rails::Engine#call&#39;
railties (8.1.3) lib/rails/railtie.rb:225:in &#39;Kernel#public_send&#39;
railties (8.1.3) lib/rails/railtie.rb:225:in &#39;Rails::Railtie.method_missing&#39;
actionpack (8.1.3) lib/action_dispatch/routing/mapper.rb:33:in &#39;block in &lt;class:Constraints&gt;&#39;
actionpack (8.1.3) lib/action_dispatch/routing/mapper.rb:62:in &#39;ActionDispatch::Routing::Mapper::Constraints#serve&#39;
actionpack (8.1.3) lib/action_dispatch/journey/router.rb:35:in &#39;block in ActionDispatch::Journey::Router#serve&#39;
actionpack (8.1.3) lib/action_dispatch/journey/router.rb:92:in &#39;block in ActionDispatch::Journey::Router#recognize&#39;
actionpack (8.1.3) lib/action_dispatch/journey/router.rb:66:in &#39;Array#each&#39;
actionpack (8.1.3) lib/action_dispatch/journey/router.rb:66:in &#39;ActionDispatch::Journey::Router#recognize&#39;
actionpack (8.1.3) lib/action_dispatch/journey/router.rb:31:in &#39;ActionDispatch::Journey::Router#serve&#39;
actionpack (8.1.3) lib/action_dispatch/routing/route_set.rb:906:in &#39;ActionDispatch::Routing::RouteSet#call&#39;
railties (8.1.3) lib/rails/engine/lazy_route_set.rb:60:in &#39;Rails::Engine::LazyRouteSet#call&#39;
railties (8.1.3) lib/rails/engine.rb:534:in &#39;Rails::Engine#call&#39;
railties (8.1.3) lib/rails/railtie.rb:225:in &#39;Kernel#public_send&#39;
railties (8.1.3) lib/rails/railtie.rb:225:in &#39;Rails::Railtie.method_missing&#39;
actionpack (8.1.3) lib/action_dispatch/routing/mapper.rb:33:in &#39;block in &lt;class:Constraints&gt;&#39;
actionpack (8.1.3) lib/action_dispatch/routing/mapper.rb:62:in &#39;ActionDispatch::Routing::Mapper::Constraints#serve&#39;
actionpack (8.1.3) lib/action_dispatch/journey/router.rb:35:in &#39;block in ActionDispatch::Journey::Router#serve&#39;
actionpack (8.1.3) lib/action_dispatch/journey/router.rb:92:in &#39;block in ActionDispatch::Journey::Router#recognize&#39;
actionpack (8.1.3) lib/action_dispatch/journey/router.rb:66:in &#39;Array#each&#39;
actionpack (8.1.3) lib/action_dispatch/journey/router.rb:66:in &#39;ActionDispatch::Journey::Router#recognize&#39;
actionpack (8.1.3) lib/action_dispatch/journey/router.rb:31:in &#39;ActionDispatch::Journey::Router#serve&#39;
actionpack (8.1.3) lib/action_dispatch/routing/route_set.rb:906:in &#39;ActionDispatch::Routing::RouteSet#call&#39;
railties (8.1.3) lib/rails/engine/lazy_route_set.rb:60:in &#39;Rails::Engine::LazyRouteSet#call&#39;
batch-loader (2.0.6) lib/batch_loader/middleware.rb:11:in &#39;BatchLoader::Middleware#call&#39;
bullet (8.1.0) lib/bullet/rack.rb:21:in &#39;Bullet::Rack#call&#39;
omniauth (2.1.4) lib/omniauth/strategy.rb:202:in &#39;OmniAuth::Strategy#call!&#39;
omniauth (2.1.4) lib/omniauth/strategy.rb:169:in &#39;OmniAuth::Strategy#call&#39;
omniauth (2.1.4) lib/omniauth/strategy.rb:202:in &#39;OmniAuth::Strategy#call!&#39;
omniauth (2.1.4) lib/omniauth/strategy.rb:169:in &#39;OmniAuth::Strategy#call&#39;
omniauth (2.1.4) lib/omniauth/strategy.rb:202:in &#39;OmniAuth::Strategy#call!&#39;
omniauth (2.1.4) lib/omniauth/strategy.rb:169:in &#39;OmniAuth::Strategy#call&#39;
omniauth (2.1.4) lib/omniauth/strategy.rb:202:in &#39;OmniAuth::Strategy#call!&#39;
omniauth (2.1.4) lib/omniauth/strategy.rb:169:in &#39;OmniAuth::Strategy#call&#39;
omniauth (2.1.4) lib/omniauth/builder.rb:48:in &#39;OmniAuth::Builder#call&#39;
warden-jwt_auth (0.12.0) lib/warden/jwt_auth/middleware/token_dispatcher.rb:20:in &#39;Warden::JWTAuth::Middleware::TokenDispatcher#call&#39;
warden-jwt_auth (0.12.0) lib/warden/jwt_auth/middleware/revocation_manager.rb:21:in &#39;Warden::JWTAuth::Middleware::RevocationManager#call&#39;
rack (3.2.6) lib/rack/builder.rb:283:in &#39;Rack::Builder#call&#39;
warden-jwt_auth (0.12.0) lib/warden/jwt_auth/middleware.rb:22:in &#39;Warden::JWTAuth::Middleware#call&#39;
warden (1.2.9) lib/warden/manager.rb:36:in &#39;block in Warden::Manager#call&#39;
warden (1.2.9) lib/warden/manager.rb:34:in &#39;Kernel#catch&#39;
warden (1.2.9) lib/warden/manager.rb:34:in &#39;Warden::Manager#call&#39;
/home/alecslupu/Sites/decidim/develop/decidim-core/lib/decidim/middleware/strip_x_forwarded_host.rb:12:in &#39;Decidim::Middleware::StripXForwardedHost#call&#39;
/home/alecslupu/Sites/decidim/develop/decidim-core/lib/decidim/middleware/current_organization.rb:22:in &#39;Decidim::Middleware::CurrentOrganization#call&#39;
rack (3.2.6) lib/rack/tempfile_reaper.rb:20:in &#39;Rack::TempfileReaper#call&#39;
rack (3.2.6) lib/rack/etag.rb:29:in &#39;Rack::ETag#call&#39;
rack (3.2.6) lib/rack/conditional_get.rb:31:in &#39;Rack::ConditionalGet#call&#39;
rack (3.2.6) lib/rack/head.rb:15:in &#39;Rack::Head#call&#39;
actionpack (8.1.3) lib/action_dispatch/http/content_security_policy.rb:38:in &#39;ActionDispatch::ContentSecurityPolicy::Middleware#call&#39;
rack-session (2.1.2) lib/rack/session/abstract/id.rb:274:in &#39;Rack::Session::Abstract::Persisted#context&#39;
rack-session (2.1.2) lib/rack/session/abstract/id.rb:268:in &#39;Rack::Session::Abstract::Persisted#call&#39;
actionpack (8.1.3) lib/action_dispatch/middleware/cookies.rb:708:in &#39;ActionDispatch::Cookies#call&#39;
activerecord (8.1.3) lib/active_record/migration.rb:671:in &#39;ActiveRecord::Migration::CheckPending#call&#39;
actionpack (8.1.3) lib/action_dispatch/middleware/callbacks.rb:31:in &#39;block in ActionDispatch::Callbacks#call&#39;
activesupport (8.1.3) lib/active_support/callbacks.rb:101:in &#39;ActiveSupport::Callbacks#run_callbacks&#39;
actionpack (8.1.3) lib/action_dispatch/middleware/callbacks.rb:30:in &#39;ActionDispatch::Callbacks#call&#39;
actionpack (8.1.3) lib/action_dispatch/middleware/executor.rb:20:in &#39;ActionDispatch::Executor#call&#39;
actionpack (8.1.3) lib/action_dispatch/middleware/actionable_exceptions.rb:18:in &#39;ActionDispatch::ActionableExceptions#call&#39;
actionpack (8.1.3) lib/action_dispatch/middleware/debug_exceptions.rb:31:in &#39;ActionDispatch::DebugExceptions#call&#39;
web-console (4.3.0) lib/web_console/middleware.rb:132:in &#39;WebConsole::Middleware#call_app&#39;
web-console (4.3.0) lib/web_console/middleware.rb:28:in &#39;block in WebConsole::Middleware#call&#39;
web-console (4.3.0) lib/web_console/middleware.rb:17:in &#39;Kernel#catch&#39;
web-console (4.3.0) lib/web_console/middleware.rb:17:in &#39;WebConsole::Middleware#call&#39;
actionpack (8.1.3) lib/action_dispatch/middleware/show_exceptions.rb:32:in &#39;ActionDispatch::ShowExceptions#call&#39;
railties (8.1.3) lib/rails/rack/logger.rb:41:in &#39;Rails::Rack::Logger#call_app&#39;
railties (8.1.3) lib/rails/rack/logger.rb:29:in &#39;Rails::Rack::Logger#call&#39;
actionpack (8.1.3) lib/action_dispatch/middleware/remote_ip.rb:98:in &#39;ActionDispatch::RemoteIp#call&#39;
request_store (1.7.0) lib/request_store/middleware.rb:19:in &#39;RequestStore::Middleware#call&#39;
actionpack (8.1.3) lib/action_dispatch/middleware/request_id.rb:34:in &#39;ActionDispatch::RequestId#call&#39;
rack (3.2.6) lib/rack/method_override.rb:28:in &#39;Rack::MethodOverride#call&#39;
activesupport (8.1.3) lib/active_support/cache/strategy/local_cache_middleware.rb:30:in &#39;ActiveSupport::Cache::Strategy::LocalCache::Middleware#call&#39;
actionpack (8.1.3) lib/action_dispatch/middleware/server_timing.rb:61:in &#39;block in ActionDispatch::ServerTiming#call&#39;
actionpack (8.1.3) lib/action_dispatch/middleware/server_timing.rb:26:in &#39;ActionDispatch::ServerTiming::Subscriber#collect_events&#39;
actionpack (8.1.3) lib/action_dispatch/middleware/server_timing.rb:60:in &#39;ActionDispatch::ServerTiming#call&#39;
actionpack (8.1.3) lib/action_dispatch/middleware/executor.rb:20:in &#39;ActionDispatch::Executor#call&#39;
actionpack (8.1.3) lib/action_dispatch/middleware/static.rb:27:in &#39;ActionDispatch::Static#call&#39;
rack (3.2.6) lib/rack/sendfile.rb:131:in &#39;Rack::Sendfile#call&#39;
actionpack (8.1.3) lib/action_dispatch/middleware/host_authorization.rb:143:in &#39;ActionDispatch::HostAuthorization#call&#39;
rack-mini-profiler (4.0.1) lib/mini_profiler.rb:334:in &#39;Rack::MiniProfiler#call&#39;
shakapacker (9.7.0) lib/shakapacker/dev_server_proxy.rb:25:in &#39;Shakapacker::DevServerProxy#perform_request&#39;
rack-proxy (0.7.7) lib/rack/proxy.rb:87:in &#39;Rack::Proxy#call&#39;
rack-cors (3.0.0) lib/rack/cors.rb:102:in &#39;Rack::Cors#call&#39;
railties (8.1.3) lib/rails/engine.rb:534:in &#39;Rails::Engine#call&#39;
puma (7.2.0) lib/puma/configuration.rb:296:in &#39;Puma::Configuration::ConfigMiddleware#call&#39;
puma (7.2.0) lib/puma/request.rb:103:in &#39;block in Puma::Request#handle_request&#39;
puma (7.2.0) lib/puma/thread_pool.rb:355:in &#39;Puma::ThreadPool#with_force_shutdown&#39;
puma (7.2.0) lib/puma/request.rb:102:in &#39;Puma::Request#handle_request&#39;
puma (7.2.0) lib/puma/server.rb:503:in &#39;Puma::Server#process_client&#39;
puma (7.2.0) lib/puma/server.rb:262:in &#39;block in Puma::Server#run&#39;
puma (7.2.0) lib/puma/thread_pool.rb:182:in &#39;block in Puma::ThreadPool#spawn_thread&#39;

Information for cause: NoMethodError (undefined method &#39;name&#39; for nil):
  
/home/alecslupu/Sites/decidim/develop/decidim-blogs/app/helpers/decidim/blogs/posts_select_helper.rb:12:in &#39;Decidim::Blogs::PostsSelectHelper#fo_post_author_select_field&#39;
/home/alecslupu/Sites/decidim/develop/decidim-blogs/app/views/decidim/blogs/posts/_form.html.erb:2
actionview (8.1.3) lib/action_view/base.rb:281:in &#39;Kernel#public_send&#39;
actionview (8.1.3) lib/action_view/base.rb:281:in &#39;ActionView::Base#_run&#39;
actionview (8.1.3) lib/action_view/template.rb:284:in &#39;block in ActionView::Template#render&#39;
activesupport (8.1.3) lib/active_support/notifications.rb:212:in &#39;ActiveSupport::Notifications.instrument&#39;
actionview (8.1.3) lib/action_view/template.rb:589:in &#39;ActionView::Template#instrument_render_template&#39;
actionview (8.1.3) lib/action_view/template.rb:272:in &#39;ActionView::Template#render&#39;
actionview (8.1.3) lib/action_view/renderer/partial_renderer.rb:268:in &#39;block in ActionView::PartialRenderer#render_partial_template&#39;
activesupport (8.1.3) lib/active_support/notifications.rb:210:in &#39;block in ActiveSupport::Notifications.instrument&#39;
activesupport (8.1.3) lib/active_support/notifications/instrumenter.rb:58:in &#39;ActiveSupport::Notifications::Instrumenter#instrument&#39;
activesupport (8.1.3) lib/active_support/notifications.rb:210:in &#39;ActiveSupport::Notifications.instrument&#39;
actionview (8.1.3) lib/action_view/renderer/partial_renderer.rb:262:in &#39;ActionView::PartialRenderer#render_partial_template&#39;
actionview (8.1.3) lib/action_view/renderer/object_renderer.rb:31:in &#39;ActionView::ObjectRenderer#render_partial_template&#39;
actionview (8.1.3) lib/action_view/renderer/partial_renderer.rb:253:in &#39;ActionView::PartialRenderer#render&#39;
actionview (8.1.3) lib/action_view/renderer/object_renderer.rb:16:in &#39;ActionView::ObjectRenderer#render_object_with_partial&#39;
actionview (8.1.3) lib/action_view/renderer/renderer.rb:74:in &#39;ActionView::Renderer#render_partial_to_object&#39;
actionview (8.1.3) lib/action_view/renderer/renderer.rb:29:in &#39;ActionView::Renderer#render_to_object&#39;
actionview (8.1.3) lib/action_view/renderer/renderer.rb:24:in &#39;ActionView::Renderer#render&#39;
actionview (8.1.3) lib/action_view/helpers/rendering_helper.rb:145:in &#39;block in ActionView::Helpers::RenderingHelper#render&#39;
actionview (8.1.3) lib/action_view/base.rb:308:in &#39;ActionView::Base#in_rendering_context&#39;
actionview (8.1.3) lib/action_view/helpers/rendering_helper.rb:141:in &#39;ActionView::Helpers::RenderingHelper#render&#39;
/home/alecslupu/Sites/decidim/develop/decidim-blogs/app/views/decidim/blogs/posts/new.html.erb:11
actionview (8.1.3) lib/action_view/helpers/capture_helper.rb:50:in &#39;block in ActionView::Helpers::CaptureHelper#capture&#39;
actionview (8.1.3) lib/action_view/buffers.rb:75:in &#39;ActionView::OutputBuffer#capture&#39;
actionview (8.1.3) lib/action_view/helpers/capture_helper.rb:50:in &#39;ActionView::Helpers::CaptureHelper#capture&#39;
actionview (8.1.3) lib/action_view/helpers/form_helper.rb:775:in &#39;ActionView::Helpers::FormHelper#form_with&#39;
actionview (8.1.3) lib/action_view/helpers/form_helper.rb:462:in &#39;ActionView::Helpers::FormHelper#form_for&#39;
/home/alecslupu/Sites/decidim/develop/decidim-core/app/helpers/decidim/decidim_form_helper.rb:30:in &#39;Decidim::DecidimFormHelper#decidim_form_for&#39;
/home/alecslupu/Sites/decidim/develop/decidim-blogs/app/views/decidim/blogs/posts/new.html.erb:9
actionview (8.1.3) lib/action_view/helpers/capture_helper.rb:50:in &#39;block in ActionView::Helpers::CaptureHelper#capture&#39;
actionview (8.1.3) lib/action_view/buffers.rb:75:in &#39;ActionView::OutputBuffer#capture&#39;
actionview (8.1.3) lib/action_view/helpers/capture_helper.rb:50:in &#39;ActionView::Helpers::CaptureHelper#capture&#39;
actionview (8.1.3) lib/action_view/helpers/rendering_helper.rb:211:in &#39;ActionView::Helpers::RenderingHelper#_layout_for&#39;
actionview (8.1.3) lib/action_view/renderer/partial_renderer.rb:269:in &#39;block (2 levels) in ActionView::PartialRenderer#render_partial_template&#39;
/home/alecslupu/Sites/decidim/develop/decidim-core/app/views/layouts/decidim/shared/_layout_center.html.erb:14
actionview (8.1.3) lib/action_view/helpers/capture_helper.rb:50:in &#39;block in ActionView::Helpers::CaptureHelper#capture&#39;
actionview (8.1.3) lib/action_view/buffers.rb:75:in &#39;ActionView::OutputBuffer#capture&#39;
actionview (8.1.3) lib/action_view/helpers/capture_helper.rb:50:in &#39;ActionView::Helpers::CaptureHelper#capture&#39;
actionview (8.1.3) lib/action_view/helpers/tag_helper.rb:521:in &#39;ActionView::Helpers::TagHelper#content_tag&#39;
/home/alecslupu/Sites/decidim/develop/decidim-core/app/views/layouts/decidim/shared/_layout_center.html.erb:13
actionview (8.1.3) lib/action_view/base.rb:281:in &#39;Kernel#public_send&#39;
actionview (8.1.3) lib/action_view/base.rb:281:in &#39;ActionView::Base#_run&#39;
actionview (8.1.3) lib/action_view/template.rb:284:in &#39;block in ActionView::Template#render&#39;
activesupport (8.1.3) lib/active_support/notifications.rb:212:in &#39;ActiveSupport::Notifications.instrument&#39;
actionview (8.1.3) lib/action_view/template.rb:589:in &#39;ActionView::Template#instrument_render_template&#39;
actionview (8.1.3) lib/action_view/template.rb:272:in &#39;ActionView::Template#render&#39;
actionview (8.1.3) lib/action_view/renderer/partial_renderer.rb:268:in &#39;block in ActionView::PartialRenderer#render_partial_template&#39;
activesupport (8.1.3) lib/active_support/notifications.rb:210:in &#39;block in ActiveSupport::Notifications.instrument&#39;
activesupport (8.1.3) lib/active_support/notifications/instrumenter.rb:58:in &#39;ActiveSupport::Notifications::Instrumenter#instrument&#39;
activesupport (8.1.3) lib/active_support/notifications.rb:210:in &#39;ActiveSupport::Notifications.instrument&#39;
actionview (8.1.3) lib/action_view/renderer/partial_renderer.rb:262:in &#39;ActionView::PartialRenderer#render_partial_template&#39;
actionview (8.1.3) lib/action_view/renderer/partial_renderer.rb:253:in &#39;ActionView::PartialRenderer#render&#39;
actionview (8.1.3) lib/action_view/renderer/renderer.rb:78:in &#39;ActionView::Renderer#render_partial_to_object&#39;
actionview (8.1.3) lib/action_view/renderer/renderer.rb:49:in &#39;ActionView::Renderer#render_partial&#39;
actionview (8.1.3) lib/action_view/helpers/rendering_helper.rb:143:in &#39;block in ActionView::Helpers::RenderingHelper#render&#39;
actionview (8.1.3) lib/action_view/base.rb:308:in &#39;ActionView::Base#in_rendering_context&#39;
actionview (8.1.3) lib/action_view/helpers/rendering_helper.rb:141:in &#39;ActionView::Helpers::RenderingHelper#render&#39;
/home/alecslupu/Sites/decidim/develop/decidim-blogs/app/views/decidim/blogs/posts/new.html.erb:2
actionview (8.1.3) lib/action_view/base.rb:281:in &#39;Kernel#public_send&#39;
actionview (8.1.3) lib/action_view/base.rb:281:in &#39;ActionView::Base#_run&#39;
actionview (8.1.3) lib/action_view/template.rb:284:in &#39;block in ActionView::Template#render&#39;
activesupport (8.1.3) lib/active_support/notifications.rb:212:in &#39;ActiveSupport::Notifications.instrument&#39;
actionview (8.1.3) lib/action_view/template.rb:589:in &#39;ActionView::Template#instrument_render_template&#39;
actionview (8.1.3) lib/action_view/template.rb:272:in &#39;ActionView::Template#render&#39;
actionview (8.1.3) lib/action_view/renderer/template_renderer.rb:66:in &#39;block (2 levels) in ActionView::TemplateRenderer#render_template&#39;
activesupport (8.1.3) lib/active_support/notifications.rb:210:in &#39;block in ActiveSupport::Notifications.instrument&#39;
activesupport (8.1.3) lib/active_support/notifications/instrumenter.rb:58:in &#39;ActiveSupport::Notifications::Instrumenter#instrument&#39;
activesupport (8.1.3) lib/active_support/notifications.rb:210:in &#39;ActiveSupport::Notifications.instrument&#39;
actionview (8.1.3) lib/action_view/renderer/template_renderer.rb:60:in &#39;block in ActionView::TemplateRenderer#render_template&#39;
actionview (8.1.3) lib/action_view/renderer/template_renderer.rb:76:in &#39;block in ActionView::TemplateRenderer#render_with_layout&#39;
activesupport (8.1.3) lib/active_support/notifications.rb:210:in &#39;block in ActiveSupport::Notifications.instrument&#39;
activesupport (8.1.3) lib/active_support/notifications/instrumenter.rb:58:in &#39;ActiveSupport::Notifications::Instrumenter#instrument&#39;
activesupport (8.1.3) lib/active_support/notifications.rb:210:in &#39;ActiveSupport::Notifications.instrument&#39;
actionview (8.1.3) lib/action_view/renderer/template_renderer.rb:75:in &#39;ActionView::TemplateRenderer#render_with_layout&#39;
actionview (8.1.3) lib/action_view/renderer/template_renderer.rb:59:in &#39;ActionView::TemplateRenderer#render_template&#39;
actionview (8.1.3) lib/action_view/renderer/template_renderer.rb:11:in &#39;ActionView::TemplateRenderer#render&#39;
actionview (8.1.3) lib/action_view/renderer/renderer.rb:58:in &#39;ActionView::Renderer#render_template_to_object&#39;
actionview (8.1.3) lib/action_view/renderer/renderer.rb:31:in &#39;ActionView::Renderer#render_to_object&#39;
actionview (8.1.3) lib/action_view/rendering.rb:136:in &#39;block in ActionView::Rendering#_render_template&#39;
actionview (8.1.3) lib/action_view/base.rb:308:in &#39;ActionView::Base#in_rendering_context&#39;
actionview (8.1.3) lib/action_view/rendering.rb:135:in &#39;ActionView::Rendering#_render_template&#39;
actionpack (8.1.3) lib/action_controller/metal/streaming.rb:179:in &#39;ActionController::Streaming#_render_template&#39;
actionview (8.1.3) lib/action_view/rendering.rb:122:in &#39;ActionView::Rendering#render_to_body&#39;
actionpack (8.1.3) lib/action_controller/metal/rendering.rb:192:in &#39;ActionController::Rendering#render_to_body&#39;
actionpack (8.1.3) lib/action_controller/metal/renderers.rb:155:in &#39;ActionController::Renderers#render_to_body&#39;
actionpack (8.1.3) lib/abstract_controller/rendering.rb:28:in &#39;AbstractController::Rendering#render&#39;
actionpack (8.1.3) lib/action_controller/metal/rendering.rb:173:in &#39;ActionController::Rendering#render&#39;
actionpack (8.1.3) lib/action_controller/metal/instrumentation.rb:31:in &#39;block (2 levels) in ActionController::Instrumentation#render&#39;
activesupport (8.1.3) lib/active_support/benchmark.rb:17:in &#39;ActiveSupport::Benchmark.realtime&#39;
actionpack (8.1.3) lib/action_controller/metal/instrumentation.rb:31:in &#39;block in ActionController::Instrumentation#render&#39;
actionpack (8.1.3) lib/action_controller/metal/instrumentation.rb:100:in &#39;ActionController::Instrumentation#cleanup_view_runtime&#39;
activerecord (8.1.3) lib/active_record/railties/controller_runtime.rb:48:in &#39;ActiveRecord::Railties::ControllerRuntime#cleanup_view_runtime&#39;
actionpack (8.1.3) lib/action_controller/metal/instrumentation.rb:30:in &#39;ActionController::Instrumentation#render&#39;
actionpack (8.1.3) lib/action_controller/metal/implicit_render.rb:39:in &#39;ActionController::ImplicitRender#default_render&#39;
actionpack (8.1.3) lib/action_controller/metal/basic_implicit_render.rb:9:in &#39;ActionController::BasicImplicitRender#send_action&#39;
actionpack (8.1.3) lib/abstract_controller/base.rb:221:in &#39;AbstractController::Base#process_action&#39;
actionpack (8.1.3) lib/action_controller/metal/rendering.rb:199:in &#39;ActionController::Rendering#process_action&#39;
actionpack (8.1.3) lib/abstract_controller/callbacks.rb:267:in &#39;block in AbstractController::Callbacks#process_action&#39;
activesupport (8.1.3) lib/active_support/callbacks.rb:121:in &#39;block in ActiveSupport::Callbacks#run_callbacks&#39;
activesupport (8.1.3) lib/active_support/core_ext/time/zones.rb:65:in &#39;Time.use_zone&#39;
/home/alecslupu/Sites/decidim/develop/decidim-core/app/controllers/concerns/decidim/use_organization_time_zone.rb:21:in &#39;Decidim::ApplicationController#use_organization_time_zone&#39;
activesupport (8.1.3) lib/active_support/callbacks.rb:130:in &#39;block in ActiveSupport::Callbacks#run_callbacks&#39;
i18n (1.14.8) lib/i18n.rb:354:in &#39;I18n::Base#with_locale&#39;
/home/alecslupu/Sites/decidim/develop/decidim-core/app/controllers/concerns/decidim/locale_switcher.rb:25:in &#39;Decidim::ApplicationController#switch_locale&#39;
activesupport (8.1.3) lib/active_support/callbacks.rb:130:in &#39;block in ActiveSupport::Callbacks#run_callbacks&#39;
activesupport (8.1.3) lib/active_support/callbacks.rb:141:in &#39;ActiveSupport::Callbacks#run_callbacks&#39;
actionpack (8.1.3) lib/abstract_controller/callbacks.rb:266:in &#39;AbstractController::Callbacks#process_action&#39;
actionpack (8.1.3) lib/action_controller/metal/rescue.rb:36:in &#39;ActionController::Rescue#process_action&#39;
actionpack (8.1.3) lib/action_controller/metal/instrumentation.rb:76:in &#39;block in ActionController::Instrumentation#process_action&#39;
activesupport (8.1.3) lib/active_support/notifications.rb:210:in &#39;block in ActiveSupport::Notifications.instrument&#39;
activesupport (8.1.3) lib/active_support/notifications/instrumenter.rb:58:in &#39;ActiveSupport::Notifications::Instrumenter#instrument&#39;
activesupport (8.1.3) lib/active_support/notifications.rb:210:in &#39;ActiveSupport::Notifications.instrument&#39;
actionpack (8.1.3) lib/action_controller/metal/instrumentation.rb:75:in &#39;ActionController::Instrumentation#process_action&#39;
actionpack (8.1.3) lib/action_controller/metal/params_wrapper.rb:259:in &#39;ActionController::ParamsWrapper#process_action&#39;
activerecord (8.1.3) lib/active_record/railties/controller_runtime.rb:39:in &#39;ActiveRecord::Railties::ControllerRuntime#process_action&#39;
actionpack (8.1.3) lib/abstract_controller/base.rb:154:in &#39;AbstractController::Base#process&#39;
actionview (8.1.3) lib/action_view/rendering.rb:40:in &#39;ActionView::Rendering#process&#39;
actionpack (8.1.3) lib/action_controller/metal.rb:252:in &#39;ActionController::Metal#dispatch&#39;
actionpack (8.1.3) lib/action_controller/metal.rb:335:in &#39;ActionController::Metal.dispatch&#39;
actionpack (8.1.3) lib/action_dispatch/routing/route_set.rb:65:in &#39;ActionDispatch::Routing::RouteSet::Dispatcher#dispatch&#39;
actionpack (8.1.3) lib/action_dispatch/routing/route_set.rb:50:in &#39;ActionDispatch::Routing::RouteSet::Dispatcher#serve&#39;
actionpack (8.1.3) lib/action_dispatch/journey/router.rb:35:in &#39;block in ActionDispatch::Journey::Router#serve&#39;
actionpack (8.1.3) lib/action_dispatch/journey/router.rb:86:in &#39;block in ActionDispatch::Journey::Router#recognize&#39;
actionpack (8.1.3) lib/action_dispatch/journey/router.rb:66:in &#39;Array#each&#39;
actionpack (8.1.3) lib/action_dispatch/journey/router.rb:66:in &#39;ActionDispatch::Journey::Router#recognize&#39;
actionpack (8.1.3) lib/action_dispatch/journey/router.rb:31:in &#39;ActionDispatch::Journey::Router#serve&#39;
actionpack (8.1.3) lib/action_dispatch/routing/route_set.rb:906:in &#39;ActionDispatch::Routing::RouteSet#call&#39;
railties (8.1.3) lib/rails/engine.rb:534:in &#39;Rails::Engine#call&#39;
railties (8.1.3) lib/rails/railtie.rb:225:in &#39;Kernel#public_send&#39;
railties (8.1.3) lib/rails/railtie.rb:225:in &#39;Rails::Railtie.method_missing&#39;
actionpack (8.1.3) lib/action_dispatch/routing/mapper.rb:33:in &#39;block in &lt;class:Constraints&gt;&#39;
actionpack (8.1.3) lib/action_dispatch/routing/mapper.rb:62:in &#39;ActionDispatch::Routing::Mapper::Constraints#serve&#39;
actionpack (8.1.3) lib/action_dispatch/journey/router.rb:35:in &#39;block in ActionDispatch::Journey::Router#serve&#39;
actionpack (8.1.3) lib/action_dispatch/journey/router.rb:92:in &#39;block in ActionDispatch::Journey::Router#recognize&#39;
actionpack (8.1.3) lib/action_dispatch/journey/router.rb:66:in &#39;Array#each&#39;
actionpack (8.1.3) lib/action_dispatch/journey/router.rb:66:in &#39;ActionDispatch::Journey::Router#recognize&#39;
actionpack (8.1.3) lib/action_dispatch/journey/router.rb:31:in &#39;ActionDispatch::Journey::Router#serve&#39;
actionpack (8.1.3) lib/action_dispatch/routing/route_set.rb:906:in &#39;ActionDispatch::Routing::RouteSet#call&#39;
railties (8.1.3) lib/rails/engine.rb:534:in &#39;Rails::Engine#call&#39;
railties (8.1.3) lib/rails/railtie.rb:225:in &#39;Kernel#public_send&#39;
railties (8.1.3) lib/rails/railtie.rb:225:in &#39;Rails::Railtie.method_missing&#39;
actionpack (8.1.3) lib/action_dispatch/routing/mapper.rb:33:in &#39;block in &lt;class:Constraints&gt;&#39;
actionpack (8.1.3) lib/action_dispatch/routing/mapper.rb:62:in &#39;ActionDispatch::Routing::Mapper::Constraints#serve&#39;
actionpack (8.1.3) lib/action_dispatch/journey/router.rb:35:in &#39;block in ActionDispatch::Journey::Router#serve&#39;
actionpack (8.1.3) lib/action_dispatch/journey/router.rb:92:in &#39;block in ActionDispatch::Journey::Router#recognize&#39;
actionpack (8.1.3) lib/action_dispatch/journey/router.rb:66:in &#39;Array#each&#39;
actionpack (8.1.3) lib/action_dispatch/journey/router.rb:66:in &#39;ActionDispatch::Journey::Router#recognize&#39;
actionpack (8.1.3) lib/action_dispatch/journey/router.rb:31:in &#39;ActionDispatch::Journey::Router#serve&#39;
actionpack (8.1.3) lib/action_dispatch/routing/route_set.rb:906:in &#39;ActionDispatch::Routing::RouteSet#call&#39;
railties (8.1.3) lib/rails/engine/lazy_route_set.rb:60:in &#39;Rails::Engine::LazyRouteSet#call&#39;
railties (8.1.3) lib/rails/engine.rb:534:in &#39;Rails::Engine#call&#39;
railties (8.1.3) lib/rails/railtie.rb:225:in &#39;Kernel#public_send&#39;
railties (8.1.3) lib/rails/railtie.rb:225:in &#39;Rails::Railtie.method_missing&#39;
actionpack (8.1.3) lib/action_dispatch/routing/mapper.rb:33:in &#39;block in &lt;class:Constraints&gt;&#39;
actionpack (8.1.3) lib/action_dispatch/routing/mapper.rb:62:in &#39;ActionDispatch::Routing::Mapper::Constraints#serve&#39;
actionpack (8.1.3) lib/action_dispatch/journey/router.rb:35:in &#39;block in ActionDispatch::Journey::Router#serve&#39;
actionpack (8.1.3) lib/action_dispatch/journey/router.rb:92:in &#39;block in ActionDispatch::Journey::Router#recognize&#39;
actionpack (8.1.3) lib/action_dispatch/journey/router.rb:66:in &#39;Array#each&#39;
actionpack (8.1.3) lib/action_dispatch/journey/router.rb:66:in &#39;ActionDispatch::Journey::Router#recognize&#39;
actionpack (8.1.3) lib/action_dispatch/journey/router.rb:31:in &#39;ActionDispatch::Journey::Router#serve&#39;
actionpack (8.1.3) lib/action_dispatch/routing/route_set.rb:906:in &#39;ActionDispatch::Routing::RouteSet#call&#39;
railties (8.1.3) lib/rails/engine/lazy_route_set.rb:60:in &#39;Rails::Engine::LazyRouteSet#call&#39;
batch-loader (2.0.6) lib/batch_loader/middleware.rb:11:in &#39;BatchLoader::Middleware#call&#39;
bullet (8.1.0) lib/bullet/rack.rb:21:in &#39;Bullet::Rack#call&#39;
omniauth (2.1.4) lib/omniauth/strategy.rb:202:in &#39;OmniAuth::Strategy#call!&#39;
omniauth (2.1.4) lib/omniauth/strategy.rb:169:in &#39;OmniAuth::Strategy#call&#39;
omniauth (2.1.4) lib/omniauth/strategy.rb:202:in &#39;OmniAuth::Strategy#call!&#39;
omniauth (2.1.4) lib/omniauth/strategy.rb:169:in &#39;OmniAuth::Strategy#call&#39;
omniauth (2.1.4) lib/omniauth/strategy.rb:202:in &#39;OmniAuth::Strategy#call!&#39;
omniauth (2.1.4) lib/omniauth/strategy.rb:169:in &#39;OmniAuth::Strategy#call&#39;
omniauth (2.1.4) lib/omniauth/strategy.rb:202:in &#39;OmniAuth::Strategy#call!&#39;
omniauth (2.1.4) lib/omniauth/strategy.rb:169:in &#39;OmniAuth::Strategy#call&#39;
omniauth (2.1.4) lib/omniauth/builder.rb:48:in &#39;OmniAuth::Builder#call&#39;
warden-jwt_auth (0.12.0) lib/warden/jwt_auth/middleware/token_dispatcher.rb:20:in &#39;Warden::JWTAuth::Middleware::TokenDispatcher#call&#39;
warden-jwt_auth (0.12.0) lib/warden/jwt_auth/middleware/revocation_manager.rb:21:in &#39;Warden::JWTAuth::Middleware::RevocationManager#call&#39;
rack (3.2.6) lib/rack/builder.rb:283:in &#39;Rack::Builder#call&#39;
warden-jwt_auth (0.12.0) lib/warden/jwt_auth/middleware.rb:22:in &#39;Warden::JWTAuth::Middleware#call&#39;
warden (1.2.9) lib/warden/manager.rb:36:in &#39;block in Warden::Manager#call&#39;
warden (1.2.9) lib/warden/manager.rb:34:in &#39;Kernel#catch&#39;
warden (1.2.9) lib/warden/manager.rb:34:in &#39;Warden::Manager#call&#39;
/home/alecslupu/Sites/decidim/develop/decidim-core/lib/decidim/middleware/strip_x_forwarded_host.rb:12:in &#39;Decidim::Middleware::StripXForwardedHost#call&#39;
/home/alecslupu/Sites/decidim/develop/decidim-core/lib/decidim/middleware/current_organization.rb:22:in &#39;Decidim::Middleware::CurrentOrganization#call&#39;
rack (3.2.6) lib/rack/tempfile_reaper.rb:20:in &#39;Rack::TempfileReaper#call&#39;
rack (3.2.6) lib/rack/etag.rb:29:in &#39;Rack::ETag#call&#39;
rack (3.2.6) lib/rack/conditional_get.rb:31:in &#39;Rack::ConditionalGet#call&#39;
rack (3.2.6) lib/rack/head.rb:15:in &#39;Rack::Head#call&#39;
actionpack (8.1.3) lib/action_dispatch/http/content_security_policy.rb:38:in &#39;ActionDispatch::ContentSecurityPolicy::Middleware#call&#39;
rack-session (2.1.2) lib/rack/session/abstract/id.rb:274:in &#39;Rack::Session::Abstract::Persisted#context&#39;
rack-session (2.1.2) lib/rack/session/abstract/id.rb:268:in &#39;Rack::Session::Abstract::Persisted#call&#39;
actionpack (8.1.3) lib/action_dispatch/middleware/cookies.rb:708:in &#39;ActionDispatch::Cookies#call&#39;
activerecord (8.1.3) lib/active_record/migration.rb:671:in &#39;ActiveRecord::Migration::CheckPending#call&#39;
actionpack (8.1.3) lib/action_dispatch/middleware/callbacks.rb:31:in &#39;block in ActionDispatch::Callbacks#call&#39;
activesupport (8.1.3) lib/active_support/callbacks.rb:101:in &#39;ActiveSupport::Callbacks#run_callbacks&#39;
actionpack (8.1.3) lib/action_dispatch/middleware/callbacks.rb:30:in &#39;ActionDispatch::Callbacks#call&#39;
actionpack (8.1.3) lib/action_dispatch/middleware/executor.rb:20:in &#39;ActionDispatch::Executor#call&#39;
actionpack (8.1.3) lib/action_dispatch/middleware/actionable_exceptions.rb:18:in &#39;ActionDispatch::ActionableExceptions#call&#39;
actionpack (8.1.3) lib/action_dispatch/middleware/debug_exceptions.rb:31:in &#39;ActionDispatch::DebugExceptions#call&#39;
web-console (4.3.0) lib/web_console/middleware.rb:132:in &#39;WebConsole::Middleware#call_app&#39;
web-console (4.3.0) lib/web_console/middleware.rb:28:in &#39;block in WebConsole::Middleware#call&#39;
web-console (4.3.0) lib/web_console/middleware.rb:17:in &#39;Kernel#catch&#39;
web-console (4.3.0) lib/web_console/middleware.rb:17:in &#39;WebConsole::Middleware#call&#39;
actionpack (8.1.3) lib/action_dispatch/middleware/show_exceptions.rb:32:in &#39;ActionDispatch::ShowExceptions#call&#39;
railties (8.1.3) lib/rails/rack/logger.rb:41:in &#39;Rails::Rack::Logger#call_app&#39;
railties (8.1.3) lib/rails/rack/logger.rb:29:in &#39;Rails::Rack::Logger#call&#39;
actionpack (8.1.3) lib/action_dispatch/middleware/remote_ip.rb:98:in &#39;ActionDispatch::RemoteIp#call&#39;
request_store (1.7.0) lib/request_store/middleware.rb:19:in &#39;RequestStore::Middleware#call&#39;
actionpack (8.1.3) lib/action_dispatch/middleware/request_id.rb:34:in &#39;ActionDispatch::RequestId#call&#39;
rack (3.2.6) lib/rack/method_override.rb:28:in &#39;Rack::MethodOverride#call&#39;
activesupport (8.1.3) lib/active_support/cache/strategy/local_cache_middleware.rb:30:in &#39;ActiveSupport::Cache::Strategy::LocalCache::Middleware#call&#39;
actionpack (8.1.3) lib/action_dispatch/middleware/server_timing.rb:61:in &#39;block in ActionDispatch::ServerTiming#call&#39;
actionpack (8.1.3) lib/action_dispatch/middleware/server_timing.rb:26:in &#39;ActionDispatch::ServerTiming::Subscriber#collect_events&#39;
actionpack (8.1.3) lib/action_dispatch/middleware/server_timing.rb:60:in &#39;ActionDispatch::ServerTiming#call&#39;
actionpack (8.1.3) lib/action_dispatch/middleware/executor.rb:20:in &#39;ActionDispatch::Executor#call&#39;
actionpack (8.1.3) lib/action_dispatch/middleware/static.rb:27:in &#39;ActionDispatch::Static#call&#39;
rack (3.2.6) lib/rack/sendfile.rb:131:in &#39;Rack::Sendfile#call&#39;
actionpack (8.1.3) lib/action_dispatch/middleware/host_authorization.rb:143:in &#39;ActionDispatch::HostAuthorization#call&#39;
rack-mini-profiler (4.0.1) lib/mini_profiler.rb:334:in &#39;Rack::MiniProfiler#call&#39;
shakapacker (9.7.0) lib/shakapacker/dev_server_proxy.rb:25:in &#39;Shakapacker::DevServerProxy#perform_request&#39;
rack-proxy (0.7.7) lib/rack/proxy.rb:87:in &#39;Rack::Proxy#call&#39;
rack-cors (3.0.0) lib/rack/cors.rb:102:in &#39;Rack::Cors#call&#39;
railties (8.1.3) lib/rails/engine.rb:534:in &#39;Rails::Engine#call&#39;
puma (7.2.0) lib/puma/configuration.rb:296:in &#39;Puma::Configuration::ConfigMiddleware#call&#39;
puma (7.2.0) lib/puma/request.rb:103:in &#39;block in Puma::Request#handle_request&#39;
puma (7.2.0) lib/puma/thread_pool.rb:355:in &#39;Puma::ThreadPool#with_force_shutdown&#39;
puma (7.2.0) lib/puma/request.rb:102:in &#39;Puma::Request#handle_request&#39;
puma (7.2.0) lib/puma/server.rb:503:in &#39;Puma::Server#process_client&#39;
puma (7.2.0) lib/puma/server.rb:262:in &#39;block in Puma::Server#run&#39;
puma (7.2.0) lib/puma/thread_pool.rb:182:in &#39;block in Puma::ThreadPool#spawn_thread&#39;
```

### Extra data

- Device:
- Device OS:
- Browser:
- Decidim Version: 0.32.0.dev
- Decidim installation: nightly 


### Additional context

_No response_

--- a/decidim-blogs/app/controllers/decidim/blogs/posts_controller.rb
+++ b/decidim-blogs/app/controllers/decidim/blogs/posts_controller.rb
@@ -12,6 +12,7 @@ module Decidim
       include Decidim::FormFactory
 
       helper_method :posts, :post, :post_presenter, :paginate_posts, :posts_most_commented, :tabs, :panels
+      before_action :authenticate_user!, only: [:new, :create, :edit, :update, :destroy]
 
       def index; end
 

--- a/decidim-blogs/spec/controllers/decidim/blogs/posts_controller_spec.rb
+++ b/decidim-blogs/spec/controllers/decidim/blogs/posts_controller_spec.rb
@@ -5,21 +5,24 @@ require "spec_helper"
 module Decidim
   module Blogs
     describe PostsController do
+      include Decidim::Core::Engine.routes.url_helpers
+
+      let(:organization) { create(:organization) }
+      let(:participatory_process) { create(:participatory_process, organization:) }
+      let!(:post_component) { create(:post_component, participatory_space: participatory_process) }
+      let!(:published) { create(:post, component: post_component, created_at: 2.days.ago, published_at: 2.days.ago) }
+
+      before do
+        request.env["decidim.current_organization"] = organization
+        request.env["decidim.current_component"] = post_component
+        request.env["decidim.current_participatory_space"] = participatory_process
+      end
+
       describe "show" do
         context "when the post has not published yet" do
-          let(:organization) { create(:organization) }
-          let(:participatory_process) { create(:participatory_process, organization:) }
-          let!(:post_component) { create(:post_component, participatory_space: participatory_process) }
           let!(:unpublished) { create(:post, component: post_component, created_at: 2.days.ago, published_at: 2.days.from_now) }
-          let!(:published) { create(:post, component: post_component, created_at: 2.days.ago, published_at: 2.days.ago) }
           let!(:another_published) { create(:post, component: post_component, created_at: 3.days.ago, published_at: 1.day.ago) }
           let!(:current_user) { create(:user, :admin, :confirmed, organization:) }
-
-          before do
-            request.env["decidim.current_organization"] = organization
-            request.env["decidim.current_component"] = post_component
-            request.env["decidim.current_participatory_space"] = participatory_process
-          end
 
           it "lists only published posts" do
             get :index
@@ -60,6 +63,57 @@ module Decidim
               get :show, params: { id: unpublished.id }
               expect(response).to have_http_status(:ok)
             end
+          end
+        end
+      end
+
+      describe "#destroy" do
+        context "when user is not authenticated" do
+          it "redirects to login page" do
+            delete(:destroy, params: { id: published.id })
+            expect(flash[:alert]).to eq("You need to log in or create an account before continuing.")
+            expect(response).to redirect_to(new_user_session_path)
+            expect(published.reload).to be_present
+          end
+        end
+      end
+
+      describe "#new" do
+        context "when user is not authenticated" do
+          it "redirects to login page" do
+            get :new
+            expect(flash[:alert]).to eq("You need to log in or create an account before continuing.")
+            expect(response).to redirect_to(new_user_session_path)
+          end
+        end
+      end
+
+      describe "#create" do
+        context "when user is not authenticated" do
+          it "redirects to login page" do
+            post(:create, params: { component_id: post_component.id })
+            expect(flash[:alert]).to eq("You need to log in or create an account before continuing.")
+            expect(response).to redirect_to(new_user_session_path)
+          end
+        end
+      end
+
+      describe "#edit" do
+        context "when user is not authenticated" do
+          it "redirects to login page" do
+            get :edit, params: { component_id: post_component.id, id: published.id }
+            expect(flash[:alert]).to eq("You need to log in or create an account before continuing.")
+            expect(response).to redirect_to(new_user_session_path)
+          end
+        end
+      end
+
+      describe "#update" do
+        context "when user is not authenticated" do
+          it "redirects to login page" do
+            put :update, params: { component_id: post_component.id, id: published.id }
+            expect(flash[:alert]).to eq("You need to log in or create an account before continuing.")
+            expect(response).to redirect_to(new_user_session_path)
           end
         end
       end

